cli/src/command/account.rs

```diff
@@ -283,6 +283,46 @@ pub async fn burn_nft_command(account_handle: &AccountHandle, nft_id: String) ->
 pub async fn balance_command(account_handle: &AccountHandle) -> Result<(), Error> {
     println_log_info!("{:#?}", account_handle.balance().await?);
 
+    let balance = account_handle.balance().await?;
+
+    let current_time = iota_sdk::utils::unix_timestamp_now().as_secs() as u32;
+
+    if balance.potentially_locked_outputs().is_empty() {
+        println_log_info!("no locked outputs");
+    } else {
+        for (output_id, can_be_unlocked) in balance.potentially_locked_outputs() {
+            if *can_be_unlocked {
+                if let Some(output_data) = account_handle.get_output(output_id).await {
+                    let output = output_data.output;
+
+                    if output.is_nft() {
+                        println_log_info!("Basic output {} can be claimed", output_id);
+                    } else {
+                        println_log_info!("NFT output {} can be claimed", output_id);
```

Comment:
Needs to be changed around :trollface: Edited my message yesterday

Reply:
you're reviewing a draft Thoralf :wink: ... you know that you have to undraft by clicking on the "READY to review" button :joy:

Reply:
Yes I know, just wanted to clarify so you can't complain that it was my fault :trollface: